core/lib/multivm/src/versions/vm_latest/tracers/refunds.rs

```diff
@@ -116,6 +116,9 @@ impl<S> RefundsTracer<S> {
 
         // For now, bootloader charges only for base fee.
         let effective_gas_price = get_batch_base_fee(&self.l1_batch);
+        if effective_gas_price == 0 {
```

Comment:
Talked in private: it is a breaking change, but most likely it is fine. Still, it is better to be able to somehow check that the partners (and Era itself) are not affected

Reply:
Hey @StanislavBreadless 
* What if we safeguard it behind a feature toggle? 
* What is the scenario that we need to check for?